src/main/java/com/eerussianguy/blazemap/mixin/RubidiumCompatMixin.java

```diff
@@ -11,7 +11,7 @@
 @Mixin(ChunkRenderList.class)
 public class RubidiumCompatMixin {
 
-    @Inject(method = "add", at = @At("HEAD"))
+    @Inject(method = "add", at = @At("HEAD"), remap = false)
```

Comment:
Rubidium, not being a Minecraft class, isn't obfuscated, and therefore its mixin shouldn't be remapped

Reply:
IIRC remapping should just skip it, but it's a good call regardless.

Reply:
Compiler said no, so adding `refmap = false` to fix it 🤷‍♀️